js/profile-w3c-common.js

```diff
@@ -79,6 +79,7 @@ define([
   "./core/highlight-vars",
   "./core/data-type",
   "./core/algorithms",
+  "./core/anchor-expander.js",
```

Comment:
Does this work? AFAIK requirejs refuses to work with `.js` file extension for relative paths.

Reply:
no, this will break :) I was testing only against w3c-respec profile. Fixing.